crux-lucene/src/crux/lucene/multi_field.clj

```diff
@@ -54,11 +54,9 @@
         search-results))
 
 (defmethod q/pred-args-spec 'lucene-text-search [_]
-  (s/cat :pred-fn #{'lucene-text-search} :args (s/spec (s/cat :query (some-fn string? symbol?) :bindings (s/* :crux.query/binding))) :return (s/? :crux.query/binding)))
```

Comment:
I'm fairly sure `:crux.query/binding` wasn't right here, since it covers more than just logic-vars.